src/ApplicationConfigInjectionTrait.php

```diff
@@ -196,36 +193,6 @@ public function injectRoutesFromConfig(array $config = null)
     }
 
     /**
-     * Create and return the pipeline map callback.
-     *
-     * The returned callback has the signature:
-     *
-     * <code>
-     * function ($item) : callable|string
-     * </code>
-     *
-     * It is suitable for mapping pipeline middleware representing the application
-     * routing o dispatching middleware to a callable; if the provided item does not
-     * match either, the item is returned verbatim.
-     *
-     * @return callable
-     */
-    private function createPipelineMapper()
```

Comment:
This functionality could be removed as `pipe()` now uses `prepareMiddleware()` throughout, and recognizes the `*_MIDDLEWARE` constants properly.